### ISSUE TITLE
Add rules to map from non-combining to combining accents

### ISSUE THREAD

**Issue by fred-wang** [mathml-refresh/mathml#11]
Original thread: https://lists.w3.org/Archives/Public/www-math/2018Mar/0001.html

I think such a rule should be in MathML Core.

**Comment by fred-wang** [comment]
David did https://mathml-refresh.github.io/mathml-core/#comb-equiv

**Comment by fred-wang** [comment]
However, I think the MathML Core text still has to be adjusted and refer to that table.

**Comment by davidcarlisle** [comment]
@fred-wang  yes in fact I had never really extracted or looked at that data from unicode.xml since adding it during the email discussion, so I just added more or less raw tables into that appendix. To be decided if it needs an appendix or should just be a table in the main body, also I am not sure we need to display the "reverse direction" table. But at least the raw data is now there.

**Comment by fred-wang** [comment]
@davidcarlisle where is the latest version of unicode.xml ; http://www.w3.org/2003/entities/2007xml/unicode.xml seems not up-to-date?

**Comment by fred-wang** [comment]
https://github.com/mathml-refresh/xml-entities I guess

**Comment by davidcarlisle** [comment]
> https://github.com/mathml-refresh/xml-entities I guess

which is the same as the one in the W3C github at

https://github.com/w3c/xml-entities/

The one in W3C TR  space is older

**Comment by fred-wang** [comment]
I think it would be good to have one "official" version to use/edit.

**Comment by davidcarlisle** [comment]
Sure but it is same as mathml-full spec (and potentially core as well) we are using the mathml-refresh github as that gives full admin control and we can generate new repositories and give write access as required, but if we move to a REC trac process (I think) we will need to move to a W3C organisation github, however I am assuming a workflow that doesn't edit on on the w3c repository we just periodically merge in changes from mathml-refresh

So historically all edits to unicode.xml for several years have been at https://github.com/w3c/xml-entities/ but I think the current policy should be that edits happen in mathml-refresh and are then merged across to https://github.com/w3c/xml-entities/

**Comment by fred-wang** [comment]
For MathML Core, I think we only want non-combining => combining  mapping.
And actually an **ordered** list of combining characters so that browsers can try them in a determined order.
I'll try to update the spec for that.

**Comment by fred-wang** [comment]
I had set https://mathml-refresh.github.io/mathml-core/#combining-character-equivalences to "non-normative" ; we should defer the details for accents to a next version of core.
